.editorconfig

```diff
@@ -17,3 +17,9 @@ charset = utf-8
 end_of_line = lf
 
 dotnet_diagnostic.CS1591.severity = silent
+
+[*.cs]
+file_header_template = Copyright (C) 2015-2024 The Neo Project.\n\n{fileName} file belongs to the neo project and is free\nsoftware distributed under the MIT software license, see the\naccompanying file LICENSE in the main directory of the\nrepository or http://www.opensource.org/licenses/mit-license.php\nfor more details.\n\nRedistribution and use in source and binary forms with or without\nmodifications are permitted.
```

Comment:
I think that is better to don't mention the filename just as before

```
// The neo is free software distributed under the MIT software license, 
// see the accompanying file LICENSE in the main directory of the
// project or http://www.opensource.org/licenses/mit-license.php 
```

Reply:
It's up to you which one you want. It allow files **ONLY** to be under the license and not everything else. Because i don't think you want your Icons/Logos to be under the license? A long with any invented algorithms or other work. Seeing how the project is copyrighted, that would allow people to steal your copyright or trademarks. Because copyright prevents people from stealing your work like anything you have invented, so they can't patent your work. But they use your source code to make or modify to their liking.